config.yml

```diff
@@ -83,6 +83,7 @@ development:
   azure:
     account: jungle
   app:
+    awsInstancePubkey: 'fake-pubkey'
```

Comment:
i don't know if it's OK to use a fake key in this format, a valid-shaped key should be in user-config.yml instead.